### PR TITLE
New version: MusicBrainz.Picard version 2.70000.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37975)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/m/MusicBrainz/Picard/2.70000.0/MusicBrainz.Picard.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: MusicBrainz.Picard
+PackageVersion: 2.70000.0
```
Comment (ItzLevvie) [inline manifests/m/MusicBrainz/Picard/2.70000.0/MusicBrainz.Picard.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.7.0.0
```

File: manifests/m/MusicBrainz/Picard/2.70000.0/MusicBrainz.Picard.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: MusicBrainz.Picard
+PackageVersion: 2.70000.0
```
Comment (ItzLevvie) [inline manifests/m/MusicBrainz/Picard/2.70000.0/MusicBrainz.Picard.yaml:5]:
```suggestion
PackageVersion: 2.7.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
Did they change their versioning scheme? Or is it just this one?

image

**Pull request merged**